styles.css

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+body {
+  background-color: ;
```

Comment:
I think we try to avoid empty commands, no?

Reply:
Thank you! I forgot this one :D